<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Email:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>emailLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="emailLineEdit">
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>passwdLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwdLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Save Email and Passwd</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>&amp;Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>&amp;Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emailLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwdLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
